common/utils.go

```diff
@@ -0,0 +1,43 @@
+// This file is part of MinIO Kubernetes Cloud
```

Comment:
What is the purpose of this package?

Actually the functionality here is not worth a separate package.
In particular it should just be:
```
b := make([]byte, size)
if _, err := io.ReadFull(rand.Reader, b); err != nil {
     // TODO: error handling
}
// now use b or encode it as base64 / hex / ...
```

Reply:
so in this package I'm planning to put all code that is being shared among portal and cluster packages to avoid copy pasting code.

Reply:
I've modified the function with the suggestion. Still since this function is being used in m3/cluster and m3/portal that's why needs to be in separate package to avoid duplicate code. What do you suggest instead of creating a new package?

Reply:
Ok I just removed lol, since m3/cluster for now will not need it anymore.